labs/a11.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Lab 11 - Security
+layout: lab
+---
+
+## Overview
+This lab is a bit unique compared to the rest, very little programming is
+involved. 
+
+## Getting help
+If you want any help with any part of this lab, join the OCF slack
+(<https://ocf.io/slack>), and mention me (slack: @mdcha, IRC: dongkyun) in
+ #decal-general.
+
+## Part 1: Threat Modeling
+Read the below prompts and answer any related questions. These questions may
+take some research and googling. Remember:
+1. What are you protecting?
+2. Who are your adversaries?
+3. How likely is it you will need to protect it?
+4. What are the consequences of failing to protect it?
+5. How many resources should you devote to protecting it?
+
+### Example: Safekeeping
+This one is just for reference, you don't have to answer it:
+
+You run a safe storage facility for customers to store a variety of valuables.
+Describe your threat model.
+
+In this scenario, you're responsible for your clients' valuables. There are a
+multitude of adversaries, including burglars ranging from smash-and-grab to more
+sophisticated attackers, disgruntled employees, clients looking to claim
+fradulent loss, and natural disasters such as earthquakes, fires, or tornados.
+
+Storage facilties containing potentially high amounts of valuables in close
+proximity to each other may serve as enticing targets for attackers and failing
+to protect the facility would result in a loss of trust that would be
+devastating for a business. 
+
+In order to protect against burglars, an variety of protections can be enabled
+from increased surveillance to 24/7 guards. The rational amount of security
+depends on the level of desired profit and likelihood of a break-in.
+
+Allowing clients to install their own security protects allows them protection
+agaisnt both disgruntled employees and acts as a protection mechanism against
+fradulent clients. 
+
+### Illicit Activity
+You're a drug dealer that communicates to your clients over SMS and offers to
+take Venmo. You usually offer to meet them at a local park only a few blocks
+away from your apartment. 
+
+First create a threat model and then determine how to go about your business
+more securely. 
+
+Secondly, if you were a fed, what tools do you have at your disposal, and how
+could you go about building a case against the above drug dealer.
+
+### Subversive Journalism
+You're a journalist criticizing the local government and trying to get your
+story to ProPublica. Describe your threat model and how you would safely deliver
+the information.
```

Comment:
might be helpful to add information about your government? for instance, if I was a journalist criticizing the Berkeley government I wouldn't give a fuck.